### DISCUSSION TITLE
Handbrake install fails because windowsdesktop-runtime-lts requires admin privileges

### DISCUSSION THREAD

**Discussion by southisup** [ScoopInstaller/Extras#9218] [Q&A] [answered]
PS C:\Users\user> `scoop install handbrake`
_Installing 'windowsdesktop-runtime-lts' (6.0.8) [64bit] from extras bucket
Loading windowsdesktop-runtime-6.0.8-win-x64.exe from cache
Checking hash of windowsdesktop-runtime-6.0.8-win-x64.exe ... ok.
Running pre_install script...
ERROR Admin privileges are required._

Any suggestions appreciated

**Comment by StarsbySea** [comment, marked as answer]
#9520 seems solve this problem.
